### PR TITLE
Adding blog post explaining how Patreon money will be distributed.

### PR DESCRIPTION
**Pull request by gamedevsam**
Need feedback please.

### CODE REVIEW

**Comment by Beeblerox** [comment]
+1

File: src/documents/blog/02-HaxeFlixel-patreon.html.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+```
+title: "HaxeFlixel Patreon"
+layout: blog-post
+postDate: 11-10-2014
```
Comment (Gama11) [inline src/documents/blog/02-HaxeFlixel-patreon.html.md]: Should probably update the date? :)

**Comment by Gama11** [comment]
:+1:

**Comment by gamedevsam** [comment]
Made some updates, if you guys don't have any problems with it, I'm merging it on 5/20.

**Pull request merged**